central/notifiers/awssh/mapping.go

```diff
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/securityhub"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/securityhub/types"
```

Comment:
```suggestion
	securityhub "github.com/aws/aws-sdk-go-v2/service/securityhub/types"
```

Reply:
Oh, that would be inconsistent with other imports down there.

Reply:
Yeah, but we could do `securityhubTypes` to make it more explicit.